examples/claim-bcc.html

```diff
@@ -0,0 +1,67 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>TREZOR Claim Bitcoin Cash</title>
+    <script>
+
+    var HD_HARDENED = 0x80000000;
+
+    function getPathFromIndex(index) {
+        return getSerializedPath([
+            (44 | HD_HARDENED) >>> 0,
+            (145 | HD_HARDENED) >>> 0,
+            (index | HD_HARDENED) >>> 0
+        ]);
+    }
+
+    function getSerializedPath(path) {
+        return path.map((i) => {
+            let s = (i & ~HD_HARDENED).toString();
+            if (i & HD_HARDENED) {
+                return s + "'";
+            } else {
+                return s;
+            }
+        }).join('/');
+    }
+
+    function calculateFee(unspents){
+
+    }
+
+    window.TREZOR_POPUP_URL = 'http://localhost/popup/popup.html';
```

Comment:
examples can be run from connect.trezor.io... so as in the other examples this shouldn't be here

not a blocker, but would be nice to fix